js/apps/system/_admin/aardvark/APP/frontend/js/views/navigationView.js

```diff
@@ -160,13 +160,11 @@
           if (!timeData.error && timeData.code === 200 && timeData.time) {
             self.fetchLicenseInfo(timeData.time);
           } else {
-            self.showGetTimeError();  
-          }
-        },
-        error: function () {
-          self.showGetTimeError();
+            self.showGetTimeError();
```

Comment:
`/_admin/time` can only be accessed through `_system` database, correct?
Therefore why do that request only then if a user is logged into the system database? 
You can check that through the global accessible variable `frontendConfig`.

Reply:
Not we're sending that request every time - and we know that for most users it will fail.

Reply:
@ajurdak @Zyqsempai what do we need to proceed here?

Reply:
Check for _system database was added in a separate PR.